### DISCUSSION TITLE
Selecting data points on graph

### DISCUSSION THREAD

**Discussion by flyinghigh012** [GSI-CS-CO/chart-fx#506] [Q&A]
Is there a built-in way to select points on the graph with your cursor and modify that data set (e.g. delete the selected data from the graph)?

**Comment by RalphSteinhagen** [comment]
The editing features are still very basic but you may want to have a look at the [EditDataSetSample](https://github.com/GSI-CS-CO/chart-fx/blob/master/chartfx-samples/src/main/java/de/gsi/chart/samples/EditDataSetSample.java):

![image](https://user-images.githubusercontent.com/46007894/162686990-adf3b44b-527c-477a-b3fc-fea452b20902.png)
